data/transactions/logic/assembler_test.go

```diff
@@ -1551,6 +1558,7 @@ txna ClearStateProgramPages 0
 	}
 	ops := testProg(t, text, AssemblerMaxVersion)
 	t2, err := Disassemble(ops.Program)
+	fmt.Println(t2)
```

Comment:
You probably want to remove this, right? It'll get printed if the next line fails.